core-services/release-controller/_releases/release-ocp-4.5.json

```diff
@@ -103,6 +103,10 @@
     "azure-ovn":{
       "optional":true,
       "prowJob":{"name":"release-openshift-ocp-installer-e2e-azure-ovn-4.5"}
+    },
+    "metal-ipi":{
```

Comment:
Leave this out until we merge and get at least 3 runs.  See my interval comment about what to set it for.